File: Aula01_Ex04_%eleitores.py
votantes = 20000 
validos = 18650
brancos = 250
nulos = 1100

pValidos = (validos*100)/votantes
pBrancos = (brancos*100)/votantes
pNulos = (nulos*100)/votantes

print("Houve na cidade um percentual de {}% de votos validos, com {}% de votos em branco e {}% de votos nulo.".format(pValidos, pBrancos, pNulos))

File: Aula01_Ex04_%eleitores.R
votantes<-as.numeric(readline(prompt="Insira o total de votantes: "))
validos<-as.numeric(readline(prompt="Insira o total de votos válidos: "))
brancos<-as.numeric(readline(prompt="Insira o total de votos brancos: "))
nulos<-as.numeric(readline(prompt="Insira o total de votos nulos: "))

pValidos = (validos*100)/votantes
pBrancos = (brancos*100)/votantes
pNulos = (nulos*100)/votantes

print(paste("No total de ", votantes, ", tivemos ", pValidos, "% de votos válidos; com ", pBrancos, "% de votos em branco e", pNulos,"% de votos nulos"))
